### DISCUSSION TITLE
MoneyManagerEx WebApo

### DISCUSSION THREAD

**Discussion by valsamis-d** [moneymanagerex/moneymanagerex#4279] [Q&A]
Hello guys,

I run into questions about the MM WebApp and I have some questions :

Is there any guide on how to install and use the WebApp?

Is the DB between WebApp and Desktop app compatible?

Thank you

**Comment by vomikan** [comment]
https://github.com/moneymanagerex/mmex.web

The webapp accepts transactions and passes them to the desktop application when it is running. webapp does not store data.
